react/Table/Virtualized/index.jsx

```diff
@@ -113,14 +115,32 @@ VirtualizedTable.defaultProps = {
   selectedItems: [],
   isSelectedItem: () => {},
   onSelect: () => {},
-  onSelectAll: () => {}
+  onSelectAll: () => {},
+  onSortChange: () => {}
```

Comment:
don't need to add `onSortChange` here, we did `onSortChange?.()` in the code (or we have to remove this part to be consistent, as you wish). But I think passing empty default func was not really a good idea anyway, it forces a behavior not necessarily real regarding to the app 🤔